apps/devhub/tasks.py

```diff
@@ -180,13 +181,24 @@ def handle_upload_validation_result(results, upload_pk, annotate=True):
     now = datetime.datetime.now()
     now_ts = utc_millesecs_from_epoch(now)
     delta = now_ts - upload_start
+    statsd.timing('devhub.validation_results_processed', delta)
+
+    # Scale the upload / processing time by MB size of the package
+    # so we can normalize large XPIs which naturally take longer to validate.
+    size = storage.size(upload.path)
+    scaled_delta = None
+    if size > 0:
+        size_in_mb = Decimal(size) / Decimal(1024 * 1024)
+        scaled_delta = Decimal(delta) / size_in_mb
+        statsd.timing('devhub.validation_results_processed_per_mb',
+                      scaled_delta)
```

Comment:
We get that error now but the issue with this is we don't know why it is a big number. We could easily end up with the opposite problem here.

size in MB | validation in ms | scaled in ms/MB
--------------|-----------------------|----------------------------
0.1 | 2000 | 20000
1 | 5000 | 5000
10 | 45000 | 4500

So validating a 0.1MB add-on in 2 seconds is more than 4 times worse than a 10MB add-on in 45 seconds.